src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-                
-    There are any number of "resheader" rows that contain simple 
+
+    There are any number of "resheader" rows that contain simple

```

Comment:
if you edited this file outside of Visual Studio, can you re-save it inside VS?

I've complained to them about this before, but... you know.